keyboards/iris/keymaps/saviof/keymap.c

```diff
@@ -0,0 +1,115 @@
+#include "iris.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJ 3
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJ,
+};
+
+//Tap Dance Declarations
+enum {
+  TD_SPC_ENT = 0
+};
+
+//Tap Dance Definitions
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [TD_SPC_ENT] = ACTION_TAP_DANCE_DOUBLE(KC_SPC, KC_ENT)
+};
+
+#define KC_ KC_TRNS
+#define _______ KC_TRNS
```

Comment:
This line isn't necessary and can be safely deleted; QMK defines this automatically.

https://github.com/qmk/qmk_firmware/blob/35e76539e7257a1932ec76882892b999e21c9d4f/quantum/quantum_keycodes.h#L29-L31